sphinx/locale/bn/LC_MESSAGES/sphinx.po

```diff
@@ -11,7 +11,7 @@ msgstr ""
 "POT-Creation-Date: 2017-04-28 15:54+0000\n"
 "PO-Revision-Date: 2017-04-28 15:59+0000\n"
 "Last-Translator: Takayuki SHIMIZUKAWA <shimizukawa@gmail.com>\n"
-"Language-Team: Bengali (http://www.transifex.com/sphinx-doc/sphinx-1/language/bn/)\n"
+"Language-Team: Bengali (https://www.transifex.com/sphinx-doc/sphinx-1/language/bn/)\n"
```

Comment:
This is not needed. These .po files are generated by `transifex-client`.
Could you revert them all?

Reply:
Removed the commit from the history and force-pushed. Let me know if anything else is needed. =)